proofread installation chapter (#2374)
centreon/centreon-documentation: versioned_docs/version-23.10/installation/technical.md

* Apache web server for Centreon web interface
* MariaDB databases to store Centreon configuration parameters as well as monitoring and performance data

> * The Centreon Gorgone process is used to send monitoring configuration parameters to the remote server and to manage it

Reviewer: the monitoring configuration? Here the word configuration refers to something specific, we need to discuss it.

> * The Centreon Gorgone process is used to send monitoring configuration parameters to the remote server and to manage it

Reviewer: ```suggestion
* The Centreon Gorgone process is used to send the monitoring configuration to the remote server and to manage it
```

* A monitoring engine to collect data

> * Collected data are sent to Centreon Broker SQL using cbmod by a monitoring engine

Reviewer: ```suggestion
* Collected data are sent to Centreon Broker SQL using cbmod by the monitoring engine
```

* Centreon Broker SQL stores information in MariaDB databases and forwards them to Centreon Broker RRD
* Centreon Broker RRD generates and updates RRD files with data in order to display performance graphs

##### Poller

Many components are used to build a poller:

* A monitoring engine to collect data

> * Collected data are sent to Centreon Broker SQL using cbmod by a monitoring engine

Reviewer: ```suggestion
* Collected data are sent to Centreon Broker SQL using cbmod by the monitoring engine
```

### Remote DBMS

> The distributed architecture with remote DBMSincludes three types of entities:

Reviewer: ```suggestion
The distributed architecture with remote DBMS includes three types of entities:
```

This architecture is used for:

* Enabling load balancing across multiple remote monitoring servers
* Network stream isolation: if your monitoring architecture has to monitor a DMZ area, it is easier (and safe) to place a remote server in the DMZ network
* Providing a remote DBMS

#### Components

##### DBMS server

The DBMS server is used only to store Centreon configuration parameters as well as monitoring and performance data in MariaDB databases

##### Central Centreon server

Many components are used to build a central Centreon server:

* Apache web server for Centreon web interface
* The central Centreon server obtains configuration and collected data from the DBMS server

> * The Centreon Gorgone process is used to send monitoring configuration parameters to the remote server and to manage it

Reviewer: ```suggestion
* The Centreon Gorgone process is used to send the monitoring configuration to the remote server and to manage it
```

* A monitoring engine to collect data

> * Collected data are sent to Centreon Broker SQL using cbmod by a monitoring engine

Reviewer: ```suggestion
* Collected data are sent to Centreon Broker SQL using cbmod by the monitoring engine
```

* Apache web server for Centreon web interface
* MariaDB databases to store Centreon configuration parameters as well as monitoring and performance data

> * The Centreon Gorgone process is used to send monitoring configuration parameters to the remote server and to manage it

Reviewer: ```suggestion
* The Centreon Gorgone process is used to send the monitoring configuration to the remote server and to manage it
```

* A monitoring engine to collect data

> * Collected data are sent to Centreon Broker SQL using cbmod by a monitoring engine

Reviewer: ```suggestion
* Collected data are sent to Centreon Broker SQL using cbmod by the monitoring engine
```

* MariaDB databases to store monitoring and performance data
* The Centreon Gorgone process is used to operate on collected data
* A monitoring engine to collect data

> * Collected data are sent to Centreon Broker SQL using cbmod by a monitoring engine

Reviewer: ```suggestion
* Collected data are sent to Centreon Broker SQL using cbmod by the monitoring engine
```
